conf/application.conf

```diff
@@ -104,6 +104,11 @@ microservice {
         port = 9583
       }
 
+      pdf_generator_service {
```

Comment:
have you added this dependency to the runbook?

Reply:
What's that (runbook)?

Reply:
Done - https://confluence.tools.tax.service.gov.uk/display/BTI/BTI+Application+Service+Runbook